static/src/js/util/metaIcon.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react'
+
+import {
+  FaClock,
+  FaCogs,
+  FaCubes,
+  FaFileAlt,
+  FaGlobe,
+  FaSlidersH,
+  FaTags
+} from 'react-icons/fa'
+
+import MetaIcon from '../components/MetaIcon/MetaIcon'
+import EDSCIcon from '../components/EDSCIcon/EDSCIcon'
+
+/**
+ * Renders icons indicating customization options for access methods.
+ * Used by CollectionResultsItem and AccessMethodRadio.
+ * @param {boolean} hasSpatialSubsetting
+ * @param {boolean} hasVariables
+ * @param {boolean} hasTransforms
+ * @param {boolean} hasFormats
+ * @param {boolean} hasTemporalSubsetting
+ * @param {boolean} hasCombine
+ * @param {boolean} forAccessMethodRadio - indicates usage for AccessMethodRadio
+ */
+export function getCustomizeIcons(
```

Comment:
I think if you want this as a util we should rename this file to `getCustomizeIcons`. We already have a `static/src/js/components/MetaIcon/MetaIcon.js`